lib/stages/testDataGen.cpp

```diff
@@ -123,6 +141,9 @@ void testDataGen::main_thread() {
     double frame_length =
         samples_per_data_set * ts_to_double(Telescope::instance().seq_length()) / num_links;
 
+    if (((type == "random") || (type == "onehot")) && _seed)
```

Comment:
We should likely add a comment to the header docs to indicate a seed of 0 is basically non-seeded.

Reply:
Done